content/ros/ROS Installation/ros_catkin_installer.sh

```diff
@@ -77,6 +83,7 @@ function create_catkin_workspace {
 	catkin_make
 	echo "Sourcing the workspace"
 	source devel/setup.bash
+	sudo chown -R $SUDO_USER:$SUDO_USER ~/catkin_ws/
```

Comment:
does the workspace in `$HOME` really need to be `chown` to `$SUDO_USER`? Seems excessive.

Reply:
Same as above, why is this needed?